### ISSUE TITLE
Slow down

### ISSUE THREAD

**Issue by CaseyCZ** [KixAss/homebridge-lgtv-2012a#4]
![image](https://user-images.githubusercontent.com/25530709/114070218-10dc6280-98a0-11eb-943d-10a430428494.png)

**Comment by Jordy15675** [comment]
Yep seeing the same issue, would be good to see if there was a solution to this

**Comment by ganewate** [comment]
Hi,

I don't get the slowdown message, but i do get an error spamming constantly also talking about the active identifier characteristic if its related and helps in resolving:

[homebridge-lgtv-2012a] This plugin generated a warning from the characteristic 'Active Identifier': characteristic value expected valid finite number and received "undefined" (undefined). See https://git.io/JtMGR for more info.
